client/packages/requisitions/src/RequestRequisition/ListView/ProgramRequisitionOptions.tsx

```diff
@@ -43,7 +53,12 @@ const useProgramRequisitionOptions = (
     setPeriod(null);
   }, [orderType]);
 
-  return {
+  let allOptions: {
```

Comment:
:nit `allOptions` is not reassigned
```suggestion
  const allOptions: {
```

Reply:
Thanks, i blame rust